src/main/java/uk/gov/hmcts/reform/sscs/service/RequestTranslationService.java

```diff
@@ -101,18 +100,12 @@ public boolean sendCaseToWlu(CaseDetails<SscsCaseData> caseDetails) {
             Map<String, byte[]> map = new HashMap<>();
             for (SscsDocument doc : sscsCaseData.getSscsDocument()) {
                 if (doc.getValue().getDocumentTranslationStatus() != null
-                        && doc.getValue().getDocumentTranslationStatus().equals(SscsDocumentTranslationStatus.TRANSLATION_REQUIRED)) {
-                    if (doc.getValue().getDocumentType() != null
-                            && (doc.getValue().getDocumentType().equalsIgnoreCase("appellantEvidence")
-                            || doc.getValue().getDocumentType().equalsIgnoreCase("Decision Notice")
-                            || doc.getValue().getDocumentType().equalsIgnoreCase("Direction Notice")
-                            || doc.getValue().getDocumentType().equalsIgnoreCase("sscs1"))) {
-                        doc.getValue().setDocumentTranslationStatus(SscsDocumentTranslationStatus.TRANSLATION_REQUESTED);
```

Comment:
Was this block intended to be removed?

Reply:
We intentionally removed this block as it was agreed that there would be no restrictions on the DocumentType if a document was marked as requiring translation